std/stdio.d

```diff
@@ -4527,12 +4527,16 @@ ref File makeGlobal(alias handle)()
     static __gshared File.Impl impl;
     static __gshared File result;
     static shared bool initialized;
-    import std.concurrency : initOnce;
-    initOnce!initialized({
-        impl.handle = handle;
-        result._p = &impl;
-        return true;
-    }());
+    if (!initialized)
```

Comment:
Seems redundant, initOnce does effectively the same thing internally. If this is to help with inlining the atomicLoad, aren't we better off fixing the compiler instead of doing this "triple checked singleton"?